core/driver/sources/src/main/scala/org/virtuslab/ideprobe/ProbeDriver.scala

```diff
@@ -86,6 +86,10 @@ class ProbeDriver(
     }
   }
 
+  def refreshAllExternalProjectsAsync(project: ProjectRef): Unit = {
```

Comment:
add = ProjectRef.Default

Reply:
Done with method overloading (to make it callable from non-scala client code)